recipes/yq/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "yq" %}
+{% set version = "2.4.1" %}
+{% set sha256 = "4833d4055b0f1c1f1a2fd292421b3472da39c7dc2727d7819efd11065a5fd310" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: 
+    - python -m pip install --no-deps --ignore-installed .
+    - chmod 755 $PREFIX/bin/{y,x}q
```

Comment:
upstream is using "scripts" instead of entry-points. This is preventing the noarch builds to pass the tests.

Reply:
Is this enough to fix it? Looking at https://github.com/conda/conda-build/pull/2672/files, half of the change supposedly needed to fix it is just this, but there's also some metadata change, and I don't know whether that matters.

Reply:
I don't think the metadata issue matters. However, I ran the test again and this is what I get if I build it using `noarch: python`

```
TEST START: /data/conda-bld/noarch/yq-2.4.1-py_0.tar.bz2
Deleting work directory, /data/conda-bld/yq_1522365327801/work/yq-2.4.1
INFO:filelock:Lock 140650291855032 acquired on /opt/conda/locks/L29wdC9jb25kYS9wa2dz
INFO:filelock:Lock 140650291854976 acquired on /opt/conda/locks/L2RhdGEvY29uZGEtYmxk
INFO:filelock:Lock 140650291855032 released on /opt/conda/locks/L29wdC9jb25kYS9wa2dz
INFO:filelock:Lock 140650291854976 released on /opt/conda/locks/L2RhdGEvY29uZGEtYmxk

The following NEW packages will be INSTALLED:

    ca-certificates: 2018.1.18-0      conda-forge
    certifi:         2018.1.18-py36_0 conda-forge
    jq:              1.5-4            conda-forge
    ncurses:         5.9-10           conda-forge
    oniguruma:       6.8.0-0          conda-forge
    openssl:         1.0.2n-0         conda-forge
    pip:             9.0.1-py36_1     conda-forge
    python:          3.6.4-0          conda-forge
    pyyaml:          3.12-py36_1      conda-forge
    readline:        7.0-0            conda-forge
    setuptools:      39.0.1-py36_0    conda-forge
    sqlite:          3.20.1-2         conda-forge
    tk:              8.6.7-0          conda-forge
    wheel:           0.30.0-py36_2    conda-forge
    xmltodict:       0.11.0-py36_0    conda-forge
    xz:              5.2.3-0          conda-forge
    yaml:            0.1.7-0          conda-forge
    yq:              2.4.1-py_0       local      
    zlib:            1.2.11-0         conda-forge


WARNING: conda-build appears to be out of date. You have version 2.1.18 but the
latest version is 3.7.2. Run

conda update -n root conda-build

to get the latest version.

+ source /opt/conda/bin/activate /data/conda-bld/yq_1522365327801/_t_env
+ /data/conda-bld/yq_1522365327801/_t_env/bin/python -s /data/conda-bld/yq_1522365327801/test_tmp/run_test.py
import: 'yq'
+ /bin/bash -x -e /data/conda-bld/yq_1522365327801/test_tmp/run_test.sh
+ ls -la /data/conda-bld/yq_1522365327801/_t_env/bin/yq /data/conda-bld/yq_1522365327801/_t_env/bin/xq
-rw-rw-r--. 1 conda conda 100 Mar 29 23:15 /data/conda-bld/yq_1522365327801/_t_env/bin/xq
-rw-rw-r--. 1 conda conda 101 Mar 29 23:15 /data/conda-bld/yq_1522365327801/_t_env/bin/yq
+ yq --help
/data/conda-bld/yq_1522365327801/test_tmp/run_test.sh: line 4: /data/conda-bld/yq_1522365327801/_t_env/bin/yq: Permission denied
TESTS FAILED: yq-2.4.1-py_0.tar.bz2
```

Another option is for me to patch the source code and replace `scripts` with `entry-points`. If you want me to do that, I am happy to do it post-merge.

Reply:
Oh, I missed that this wasn't `noarch: python`. It probably could be but for the scripts issue; if it's easy to replace with entry points and make it noarch, that would be a good thing to do in the feedstock. Merging now though.